src/main/java/appeng/parts/misc/PartStorageBus.java

```diff
@@ -367,6 +368,41 @@ private void resetCache()
 		Platform.postListChanges( before, after, this, this.mySrc );
 	}
 
+	@SuppressWarnings( "unchecked" )
+	private IMEInventory<? extends IAEItemStack> getInventoryWrapper( TileEntity target )
+	{
+
+		EnumFacing targetSide = this.getSide().getFacing().getOpposite();
+
+		// Prioritize a handler to directly link to another ME network
+		IStorageMonitorableAccessor accessor = target.getCapability( Capabilities.STORAGE_MONITORABLE_ACCESSOR, targetSide );
+
+		if( accessor != null )
+		{
+			IStorageMonitorable inventory = accessor.getInventory( mySrc );
+			if( inventory != null )
+			{
+				return inventory.getItemInventory();
+			}
+
+			// So this could / can be a design decision. If the tile does support our custom capability,
+			// but it does not return an inventory for the action source, we do NOT fall back to using
+			// IItemHandler's, as that might circumvent the security setings, and might also cause
+			// performance issues.
+			return null;
+		}
+
+		// Check via cap for IItemHandler
+		IItemHandler handlerExt = target.getCapability( CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, targetSide );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'handlerExt' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)